Dockerfile

```diff
@@ -15,7 +15,9 @@ RUN ./build.sh -v
 
 
 FROM debian:stretch
-RUN apt-get update && apt-get install -y mono-devel && apt-get clean all
+RUN apt-get update && apt-get install -y mono-devel curl && apt-get clean all
+RUN curl https://ssl-ccp.godaddy.com/repository/gdig2.crt.pem >> /etc/ssl/certs/ca-certificates.crt
```

Comment:
Much better, thank you!

The only other notes I have here are:
- I'd recommend using the `-o` flag to curl to choose the output file instead of redirection. Redirection might pollute the cert chain with output from curl.
- Let's call the cert chain the same file name that godaddy is giving it. `gdig2.crt.pem`

Thanks for the contribution!

Reply:
I hope that appending with cat is fine for you.